terraform/hcl2json_test.go

```diff
@@ -531,48 +532,98 @@ block "label_one" {
 }
 
 func TestExtractVariablesSuccess(t *testing.T) {
```

Comment:
maybe a missing test here, we could have a test case with a duplicate variable in the fileContent. we should fail and get invalid hcl I believe

Reply:
Actually this is more of an error case of the hcl2json case and not the extract itself. we can also add them in a seperate PR

Reply:
terraform validate will fail. We don't need a test for this in the TestExtractVariablesFailure cases, sorry for the confusion. This was the point of my second comment, we could add one in the failure cases of hcl2json instead, up to us on how much we want to unit test edge cases.